timeDifference.java

```diff
@@ -0,0 +1,27 @@
+package Occurances;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class timeDifference {
+
+	public static void main(String[] args) throws ParseException {
+		String time1 = "11:00";
+		String time2 = "12:00";
+
+		if(time2=="00:00")
+			time2="24:00";
```

Comment:
parameterize the code means write the code is different method and call it in main